extensions/amp-analytics/0.1/vendors.js

```diff
@@ -56,6 +56,23 @@ export const ANALYTICS_CONFIG = {
       'documentCharset': 'DOCUMENT_CHARSET',
     }
   },
+  
+  'atinternet': {
+    'transport': {'beacon': false, 'xhrpost': false, 'image': true},
+    'requests': {
+      'base': 'https://${log}${domain}/?s=${site}&ts=${timestamp}&r=${screenWidth}x${screenHeight}x${screenColorDepth}&re=${availableScreenWidth}x${availableScreenHeight}',
```

Comment:
Just wanted to ask if you wanted to also transmit the URL information. For example: ${ampdocUrl} or ${canonicalUrl}.

This will make attribution a bit easier as compared to referrer-based information.